tests/rados/rados_prep.py

```diff
@@ -0,0 +1,327 @@
+        kwargs: Any other param that needs to passed
+
+    Returns: True -> pass, False -> fail
+
+    """
+    duration = kwargs.get("duration", 200)
+    if operation == "write":
+        byte_size = kwargs.get("byte_size", 4096)
+        cmd = f"sudo rados --no-log-to-stderr -b {byte_size} -p {pool_name} bench {duration} write --no-cleanup"
+        try:
+            node.shell([cmd])
+            return True
+        except Exception as err:
+            log.error(f"Error running rados bench write on pool : {pool_name}")
+            log.error(err)
+            return False
+
+    # Performing read operations
+    try:
+        cmd = f"rados --no-log-to-stderr -p {pool_name} bench {duration} seq"
+        node.shell([cmd])
+        cmd = f"rados --no-log-to-stderr -p {pool_name} bench {duration} rand"
+        node.shell([cmd])
+        return True
+    except Exception as err:
+        log.error(f"Error running rados bench write on pool : {pool_name}")
+        log.error(err)
+        return False
+
+
+def set_logging_to_file(node: CephAdmin) -> bool:
+    """
+    Enables the cluster logging into files at var/log/ceph and checks file permissions
+    Args:
+        node: Cephadm node where the commands need to be executed
+
+    Returns: True -> pass, False -> fail
+    """
+    try:
+        cmd = "ceph config set global log_to_file true"
+        node.shell([cmd])
+        cmd = "ceph config set global mon_cluster_log_to_file true"
+        node.shell([cmd])
+    except Exception:
+        log.error("Error while enabling config to log into file")
+        return False
+
+    # Sleeping for 10 seconds for files to be generated
+    time.sleep(10)
+
+    cmd = "ls -ll /var/log/ceph"
+    out, err = node.shell([cmd])
+    log.info(out)
+    regex = r"\s*([-rwx]*)\.\s+\d\s+([\w]*)\s+([\w]*)\s+[\w\s:]*(ceph[\w.]*log)"
+    perm = "-rw-------"
+    user = "ceph"
+    files = ["ceph.log", "ceph.audit.log"]
+    if re.search(regex, out):
+        match = re.findall(regex, out)
+        for val in match:
+            if not (val[0] == perm and val[1] == user and val[2] == user):
+                print(f"file permissions are not correct for file : {val[3]}")
+                return False
+            if val[3] in files:
+                files.remove(val[3])
+    if files:
+        log.error(f"Did not find the log files : {files}")
+        return False
+    return True
+
+
+def enable_email_alerts(node: CephAdmin, **kwargs) -> bool:
+    """
+    Enables the email alerts module and configures alerts to be sent
+    Args:
+        node: Cephadm node where the commands need to be executed
+        **kwargs: Any other param that needs to be set
+
+    Returns: True -> pass, False -> fail
+    """
+    try:
+        cmd = "ceph mgr module enable alerts"
+        node.shell([cmd])
+
+        cmd = "ceph config set mgr mgr/alerts/smtp_host smtp.corp.redhat.com"
+        if kwargs.get("smtp_host"):
```

Comment:
There are too many if statements... a `dict` holding the key & the value as the command might help. For example
```
cmd_map = { 
  "smtp_host": "ceph config set mgr mgr/alerts/smtp_host {smtp_host}
  ...
}

for key in kwargs:
  cmd = cmd_map.get("kwargs", None)
  if not cmd: continue
  cmd = cmd.format(key=key)
  node.shell(cmd)
```

Reply:
Added all the configurable commands as part of a dictionary, with config parameter as the Key and the command as the value.